### PR TITLE
Fixes to minor visual issues from #205

### PR DESCRIPTION
**Pull request by romansp**
This PR fixes some of the issues described in #205.

### CODE REVIEW

**Comment by romansp** [comment]
Fixed no. 2, 3, 4, 5 from #205.

no. 1 - already fixed, just requires localization mentioned in #206.
no. 5 and 6 - can be reproduced only in 'production mode' when `spaceless` filter enabled, so not sure about the fix.

**Comment by rolandgeider** [comment]
Thanks, I'll take a look at this later today

**Comment by rolandgeider** [comment]
About 6 and 7, the only way I see is manually adding a ``&nbsp;`` before the labels, this is kept even after the whitespace filter but it is definitely ugly.

Besides that, I can merge the PR

**Comment by romansp** [comment]
Totally forgot about `nbsp;`. I don't think it is _that way_ ugly :). I will add one more commit with this change, if you don't mind.

**Comment by rolandgeider** [comment]
hehehe, no it's not that ugly, only a bit :)

Sure, go ahead and add the commit

**Comment by romansp** [comment]
Adding `&nbsp;` results in two whitespace characters :cry:. I think the only way to fix it - those _ugly_ `if` clause one-liners with previous HTML tag.

If it's unacceptable I will rebase without this change.

File: wger/manager/templates/schedule/overview.html [hunk 1]
```diff
@@ -20,11 +20,7 @@
     {% for schedule in schedules %}
     <tr>
         <td>
-            <a href="{% url 'manager:schedule:view' schedule.id %}">{{ schedule.name }}</a>
-
-            {% if schedule.is_active %}
-            <span class="label label-primary">{% trans "active" %}</span>
-            {% endif  %}
+            <a href="{% url 'manager:schedule:view' schedule.id %}">{{ schedule.name }}</a>&nbsp;{% if schedule.is_active %}<span class="label label-primary">{% trans "active" %}</span>{% endif  %}
```
Comment (rolandgeider) [inline wger/manager/templates/schedule/overview.html]: Is it necessary to make the ifs one liners? Shoudn't the nbsp make space in any case? Other than that, the PR is ready

Comment (rolandgeider) [inline wger/manager/templates/schedule/overview.html]: Ah, sorry, I see it now, without the one liners there are 2 whitespaces.

I would leave the ifs the way they where, the point of the nbsp is just to make a bit of space and this is not so much that it breaks the layout or anything. Plus having a readable template is important as well! :)

**Comment by romansp** [comment]
Rebased and ready.

**Comment by rolandgeider** [comment]
Merged

**Comment by rolandgeider** [comment]
@romansp, since you have an eye for visual stuff, feel free to open issues for bigger problems or things you think should be done differently, many of the pages and overviews have simply grown over time.

**Pull request merged**